content/_changelogs/10.2.0.md

```diff
@@ -0,0 +1,40 @@
+## 10.2.0
+
+_Released 6/21/2022_
+
+**Features:**
+
+- Cypress is now available natively on Apple Silicon/M1. Addresses
+  [#19908](https://github.com/cypress-io/cypress/issues/19908).
+- Cypress is now available natively on arm64/aarch64 systems running Linux.
+  Along with this feature, the Electron dependency was updated from 18.0.4 to
+  18.3.0. Addresses [#4478](https://github.com/cypress-io/cypress/issues/4478).
+- The file name input field within the Create Spec modal is now automatically
+  focused when the modal is opened. Addresses
+  [#21865](https://github.com/cypress-io/cypress/issues/21865).
+
+**Bugfixes:**
+
+- Suppressed a misleading Chromium warning about CertVerifyProcBuiltin that
+  would appear when visiting HTTPS sites in Cypress. Fixed
+  [#22128](https://github.com/cypress-io/cypress/discussions/22128).
+- Users will be able to see the custom reason-phrase in the HTTP response if the
+  custom reason-phrase is set. Otherwise, a default status message is returned
+  based on the corresponding status code as earlier. Fixed
+  [#16973](https://github.com/cypress-io/cypress/issues/16973).
+- Fixed an issue where Cypress would crash with an EMFILE error if the process
+  ran into the file descriptor limit. Cypress will now retry file reads using
+  graceful-fs. Fixed
```

Comment:
I'd either drop mention of graceful-fs or we should expand on why this is matters. We don't typically reference dependencies in the changelog like this

Reply:
I removed the reference to graceful-fs.